File: packages/screen-brightness/android/src/main/java/io/capawesome/capacitorjs/plugins/screenbrightness/classes/results/GetBrightnessResult.java
package io.capawesome.capacitorjs.plugins.screenbrightness.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.screenbrightness.interfaces.Result;

public class GetBrightnessResult implements Result {

    private final float brightness;

    public GetBrightnessResult(float brightness) {
        this.brightness = brightness;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("brightness", brightness);
        return result;
    }
}

File: packages/screen-brightness/ios/Plugin/Classes/Results/GetBrightnessResult.swift
import Foundation
import Capacitor

@objc public class GetBrightnessResult: NSObject, Result {
    let brightness: Double

    init(brightness: Double) {
        self.brightness = brightness
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["brightness"] = brightness
        return result as AnyObject
    }
}
